doc/changes.rst

```diff
@@ -6468,7 +6468,7 @@ Build system
    wider range of platforms.
 
 -  Compatibility shims for clock\_gettime() are now correctly applied
-   where needed, allowing Varnish to build on MacOS X.
+   where needed, allowing Varnish to build on Mac OS X.
```

Comment:
I should have peeled this one to a commit named `mac os` -- note that this item predates the brand transitions between `Mac OS X` and `macOS`